spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1748,6 +1748,70 @@ trade.setCustomer("Customer1");
 staxItemWriter.write(trade);
 ----
 
+[[jsonReadingWriting]]
+=== JSON Item Readers
+
+Spring Batch provides support for reading JSON resources in the following format:
+
+[source, json]
+----
+[
+  {
+    "isin": "123",
+    "quantity": 1,
+    "price": 1.2,
+    "customer": "foo"
+  },
+  {
+    "isin": "456",
+    "quantity": 2,
+    "price": 1.4,
+    "customer": "bar"
+  }
+]
+----
+
+It is assumed that the JSON resource is an array of JSON objects corresponding to
+individual items. Spring Batch is not tied to any particular JSON library.
+
+[[JsonItemReader]]
+==== JsonItemReader
+
+The `JsonItemReader` delegate JSON parsing and binding to implementations of the
+`org.springframework.batch.item.json.JsonObjectReader` interface. This interface
+is intended to be implemented using a streaming API in order to read JSON objects
```

Comment:
Replace this line with "is intended to be implemented by using a streaming API to read JSON objects"